### PR TITLE
[Keyboard] Add Slime88

### PR DESCRIPTION
**Pull request by zykrah**
Added support for Slime88 PCB. https://github.com/zykrah/slime88.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/zykrah/slime88/info.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": false,
+        "nkro": true,
+        "backlight": false,
+        "rgblight": false,
+        "audio": false
```
Comment (zvecr) [inline keyboards/zykrah/slime88/info.json:21]:
```suggestion
        "bootmagic": true,
        "mousekey": true,
        "extrakey": true,
        "console": false,
        "command": false,
        "nkro": true
```

File: keyboards/zykrah/slime88/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  [5] = LAYOUT_all(
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                        KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,               KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                        KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                                 KC_TRNS,
+    KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                        KC_TRNS, KC_TRNS, KC_TRNS),
+
```
Comment (zvecr) [inline keyboards/zykrah/slime88/keymaps/default/keymap.c:67]:
```suggestion
```

File: keyboards/zykrah/slime88/readme.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+SLIME88
```
Comment (zvecr) [inline keyboards/zykrah/slime88/readme.md:1]: This should follow the template `data/templates/keyboard/readme.md`

**Review by zvecr — CHANGES_REQUESTED** [review]
Also `keyboards/zykrah/slime88/rules.mk` should match `data/templates/keyboard/rules.mk`

File: keyboards/zykrah/slime88/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Slime88
+
+![Slime88](https://user-images.githubusercontent.com/23428162/170985512-ec79f498-7476-4ab6-8150-877c67a25f26.png)
```
Comment (drashna) [inline keyboards/zykrah/slime88/readme.md:3]: Please rehost on imgur.

File: keyboards/zykrah/slime88/info.json [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        "nkro": true
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP2", "GP29", "GP28", "GP27", "GP11", "GP12"],
+        "cols": ["GP26", "GP25", "GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP15", "GP14", "GP13", "GP0", "GP1", "GP3"]
+    },
+    "url": "https://github.com/zykrah/slime88",
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (drashna) [inline keyboards/zykrah/slime88/info.json:27]: since this is the only layout, it should probably be renamed to just `LAYOUT`

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/zykrah/slime88/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Slime88
+
+![Slime88](https://i.imgur.com/DYNJZFH.png)
```
Comment (drashna) [inline keyboards/zykrah/slime88/readme.md:3]:
```suggestion
![Slime88](https://i.imgur.com/DYNJZFHh.png)
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/zykrah/slime88/readme.md (hunk 6 above)
Comment (zykrah) [reply keyboards/zykrah/slime88/readme.md:3]: Could I ask the reasoning behind this?

File: keyboards/zykrah/slime88/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Slime88
+
+![Slime88](https://i.imgur.com/DYNJZFHh.png)
+
+*A Hotswap Multi-layout F13 TKL PCB (h88c compatible) powered by RP2040.*
```
Comment (fauxpark) [inline keyboards/zykrah/slime88/readme.md:5]:
```suggestion
A Hotswap Multi-layout F13 TKL PCB (h88c compatible) powered by RP2040.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**